src/Common/src/Interop/Windows/sspicli/NegotiationInfoClass.cs

```diff
@@ -17,14 +17,21 @@ internal class NegotiationInfoClass
 
         internal NegotiationInfoClass(SafeHandle safeHandle, int negotiationState)
         {
+            bool globalLogEnabled = GlobalLog.IsEnabled;
```

Comment:
The pattern doesn't apply only to GlobalLog but to the larger Logging feature that @josguil has in PR. If this is helping performance then we need to open a bug to track changes in all EventSource calls, not only GlobalLog. Before we do that I'd like to clarify that we're not over-complicating things.

@vancem I thought that IsEnabled() call is already implemented such that the original pattern that we've used is the recommended one. Should we change the code to cache the value locally instead of making multiple calls?